src/NServiceBus.AcceptanceTests/Routing/When_publishing_an_interface.cs

```diff
@@ -42,7 +43,8 @@ class Context : ScenarioContext
             public Type EventTypePassedToRouting { get; set; }
         }
 
-        class Publisher : EndpointConfigurationBuilder
+        [SuppressMessage("ReSharper", "MemberCanBePrivate.Global")]
```

Comment:
I thought the same as well. At the same time, if you're not aware of why this is needed, it is easy to "fix" the issue when resharper is trying to be too helpful, causing the pain in the ASB downstream. Does it hurt to be explicit in this case and have the suppress in place?